builtin/game/item.lua

```diff
@@ -205,6 +207,16 @@ function core.get_node_drops(nodename, toolname)
 	return got_items
 end
 
+local non_player_name = "!nonplayer"
```

Comment:
Ugly workaround. Why not simply use `nil`, which directly indicates that there's no player name?